packages/settingregistry/src/settingregistry.ts

```diff
@@ -944,42 +944,46 @@ export namespace SettingRegistry {
       return false;
     });
 
-    // If a default shortcut collides with another default, warn and filter.
-    // If a shortcut has already been added by the user preferences, filter it
-    // out too (this includes shortcuts that are disabled by user preferences).
-    defaults = defaults.filter(shortcut => {
-      const { disabled } = shortcut;
+    // If a default shortcut collides with another default, warn and filter,
+    // unless one of the shortcuts is a disabling shortcut (so look through
+    // disabled shortcuts first). If a shortcut has already been added by the
+    // user preferences, filter it out too (this includes shortcuts that are
+    // disabled by user preferences).
+    defaults = [
+      ...defaults.filter(s => !!s.disabled),
+      ...defaults.filter(s => !s.disabled)
+    ].filter(shortcut => {
```

Comment:
@jasongrout any chance you remember why there is two filters, one which includes if the shortcut was disabled and another if it was not disabled?

Reply:
Nevermind, I think I see it, it is just to reorder the shortcuts. Sorry for the ping.